src/components/Steps/Steps.js

```diff
@@ -56,6 +57,7 @@ export default function Steps(props) {
       <ul
         className={`
           ${STEPS_CLASSES.STEPS}
+          ${props.layout === 'vertical' ? TABS_CLASSES.VERTICAL : ''}
```

Comment:
Can we create a generic class reusable in other components instead of using the vertical class of the tabs?

Reply:
Sure.